packages/example-snap/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  extends: ['../../.eslintrc.js'],
+
+  overrides: [
+    {
+      files: ['src/**/*.js'],
+      // TODO: Replace this with the actual snap globals
+      env: {
+        browser: true,
+      },
+      globals: {
+        wallet: true,
```

Comment:
i guess not the place for this but:

it would be nice to have `window.ethereum.request` be the interface to the provider, we could leave`registerRpcMessageHandler` on `wallet` and leave that for our internal use.

Reply:
We acknowledge this but have nothing to do about it at the moment. 🙃